lms/djangoapps/teams/static/teams/js/views/edit_team_members.js

```diff
@@ -85,7 +85,7 @@
                         function () {
                             $.ajax({
                                 type: 'DELETE',
-                                url: self.teamMembershipDetailUrl + username
+                                url: self.teamMembershipDetailUrl.concat(username, '?admin=true')
```

Comment:
@peter-fogg I've elected to keep `admin=true` as a querystring instead of using the request body as you mentioned yesterday. Is that something you feel particularly strongly about?

Discussion at https://openedx.atlassian.net/wiki/display/TNL/Team+API